### PR TITLE
SFR-2179/get_collection_list

### PR DESCRIPTION
**Pull request by ayan1229**
- This PR validates the api response for a query that returns collection list by title. 
- it also verifies that when the API is hit, it doesn't return an empty JSON body in response

### CODE REVIEW

File: tests/rest_api/test_get_collection.py [hunk 1]
```diff
@@ -19,4 +19,4 @@ def test_get_collection():
 
     response_json = response.json()
     
-    assert response_json is not None
+    assert response_json is not None
```
Comment (kylevillegas93) [inline tests/rest_api/test_get_collection.py:22]: nit: new line at eof

File: tests/rest_api/test_get_list.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        f"API call failed.\n"
+        f"Expected status code: {200}\n"
+        f"Actual status code: {response.status_code}\n"
+        f"URL: {url}\n"
+        f"Response text: {response.text[:100]}..."
+    )
+
+    response_json = response.json()
+    
+    assert response_json is not None
```
Comment (kylevillegas93) [inline tests/rest_api/test_get_list.py:22]: nit: new line at eof

File: tests/rest_api/test_get_list.py
[the remark this answers is not in the document]
Comment (ayan1229) [reply tests/rest_api/test_get_list.py:1]: renaming to test_get_collection_list

File: tests/rest_api/test_get_list.py (hunk 2 above)
Comment (ayan1229) [reply tests/rest_api/test_get_list.py:22]: do you always want to keep an empty line at the end? or no lines

Comment (kylevillegas93) [reply tests/rest_api/test_get_list.py:22]: always a new line that is empty at the end of the file!

**Pull request merged**